Inbox docs update (#692)
novuhq/docs: inbox/react/components/bell.mdx

---

> title: '<Bell />'

Author: Broke each component to it's own page

---

The `Bell` component is used to display the notification bell icon. It can be used to show the number of unread notifications.

```tsx
import { Inbox, Bell } from '@novu/react';

function Novu() {
  return (
    <Inbox
      applicationIdentifier="YOUR_APPLICATION_IDENTIFIER"
      subscriberId="YOUR_SUBSCRIBER_ID"
    >
      <Bell />
    </Inbox>
  );
}
```

### Custom Bell

You can pass custom components as children to the `Bell` component to render the custom bell icon.

```tsx
import { Inbox, Bell } from '@novu/react';
import { BellIcon } from './icons';

function Novu() {
  return (
    <Inbox>
      <Bell
        renderBell={(unreadCount) => (
          <div>
            <span>{unreadCount}</span>
            <BellIcon />
          </div>
        )}
      />
    </Inbox>
  );
}
```

### Bring your own Popover

> component allows you to bring your own popover component along with custom components.

Reviewer: ```suggestion
<Inbox/> can be mounted in your own popover component. For further customization, you can also use the `renderBell` and r`enderNotification` render props.
```

> Below is an example of how you can bring your own popover component with Radix UI.

Reviewer: ```suggestion
Below is an example of how to use `<Inbox/>` with [Radix UI](https://www.radix-ui.com/).
```
